examples/pxScene2d/src/rpc/rtRemoteServer.h

```diff
@@ -84,7 +87,14 @@ class rtRemoteServer
     bool              owner_removed;
   };
 
+  struct ClientDisconnectedCB
+  {
+      clientDisconnectedCallback func;
```

Comment:
store just these 2 values in rtRemoteClient. on remote state change you can give callback from there which would be specific to client.